core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,12 +29,13 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
-      <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
+      <div ng-if="customizationArgSpec.schema.type !== 'bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
       <div>
-        <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
+        <schema-based-editor ng-class="{'align-checkbox-left': customizationArgSpec.schema.type === 'bool'}" local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
```

Comment:
Suggest naming the class `boolean-checkbox` or similar. In general class names should describe the item and not the styling that happens to be applied to it.